integration-test/440-zoos-and-other-attractions-tourism.py

```diff
@@ -3,14 +3,14 @@
 
 class ZoosAndOtherAttractionsTourism(FixtureTest):
     def test_tourism(self):
-        # City Sculpture, Detroit
-        self._run_test(16, 17645, 24242, 358445798, 'artwork')
+        # Sinclair Oil Apatosaurus, Windsor
+        self._run_test(16, 17645, 24251, 786994983, 'artwork')
         # Walt Disney World Resort
         self._run_test(13, 2240, 3421, -1228099, 'theme_park')
         # Busch Gardens Williamsburg
         self._run_test(13, 2351, 3181, 362327591, 'theme_park')
-        # Disneyland Resort
-        self._run_test(14, 2825, 6555, -4795362, 'resort')
+        # High Hampton Resort
```

Comment:
Please restore this to **Disneyland Resort**, and update the pois.yaml and landuse.yaml files to use the new `leisure=resort` tagging that OSM changed out from under us (from `tourism=resort`).

Reply:
I changed the POIs and Landuse logic, but still update the test here.